### DISCUSSION TITLE
is it possible to enable quail when using `meow-till` or `meow-find` to jump to a unicode symbol?

### DISCUSSION THREAD

**Discussion by capfredf** [meow-edit/meow#641] [Q&A]
This might be a niche request. A lot of lean/agda programs contain math symbols and Greek letters such as `⇒→⊥`.  `isearch` supports quail quite well so that I can turn on `quail` to search for symbols. But I am spoiled by meow right now. How can I make `meow-till` and `meow-find` work with `quail`?

**Comment by DogLooksGood** [comment]
I'm afraid you can't now. What would be the expected behavior?

**Reply by capfredf** [reply]
I am attaching a screencast where I use isearch to search for the symbol `×` (a multiplication sign).

 [Screencast from 2024-10-05 11-03-29.webm](https://github.com/user-attachments/assets/7ceb79ba-d0d8-49ce-8efa-de565eeac010)

By default, when an input method is enabled for a buffer, a user is able to use that method in the isearch buffer. As shown in the example above, `[⊓]` after `I-search` indicates the Agda input method is also enabled for the minibuffer, and thus I can type `\x` to get the symbol `×`.

**Reply by DogLooksGood** [reply]
Unlike isearch, `meow-find`, `meow-till` use a different way to read input, and currently the input method is not supported.
